posthog/api/test/test_capture.py

```diff
@@ -127,7 +129,7 @@ def test_capture_event(self, kafka_produce):
                 ],
             },
         }
-        with self.assertNumQueries(1):
+        with self.assertNumQueries(2):
```

Comment:
Can we add a check that the number of requests on a second capture is back to 1?

Reply:
Can we also add a test for which ever endpoint would return the ingested_event=True to the client